README.md

```diff
@@ -377,9 +272,13 @@ client.query(predicateExpr) { [weak self] result in
 
```

Comment:
I was confused by the mix between the "DecodableValue" usage and the pagination example. 

I would dedicate an entire section to the pagination example + refactoring (if it's deemed valuable but based on past comments by @erickpintor I don't think it is..)

Apart from that, in the line quoted below `either` has a typo, and the example doesn't show how to use Field type so I'd avoid mentioning it at this point

> "Now we can decode any value instance into a `BlogPost` by ether using `Value`'s `get` method or `Field` type...